test/studies/champs/functions.bash

```diff
@@ -71,14 +94,21 @@ function test_compile() {
   fi
   test_end
 
-  $CHPL_HOME/util/test/computePerfStats comp-time-$kind $CHPL_TEST_PERF_DIR/$CHPL_TEST_PERF_DESCRIPTION $CHAMPS_GRAPH_PATH/comp-time.perfkeys $kind.comp.out.tmp
-  if [[ $? -ne 0 ]] ; then
-    log_fatal_error "computing compile time stats for ${kind}"
+  # don't leave this environment set
+  if [ ! -z "$CHAMPS_QUICKSTART" ]; then
+    unset CHPL_STOP_AFTER_PASS
```

Comment:
If you really want this to be robust you should capture what `CHPL_STOP_AFTER_PASS` is prior to your export above and then reset it to that. (it's not necessarily the case that the user ran `sub_test` with `CHPL_STOP_AFTER_PASS` unset).

Reply:
Note that what I have in this PR as is doesn't change `CHPL_STOP_AFTER_PASS` if it was set already. But you're right that we are unsetting it maybe too aggressively here. In any case I already put a fix in for this. 8b4236362c